tests/unit_tests/test_workflow.py

```diff
@@ -206,16 +206,26 @@ def test_calibration_manager_save_load():
     os.remove(temp_path)
 
 
-OPTIMIZATION_TEST_DATA = {
-    "fwhm_cycles": 5,
-    "fwhm_mobility": 0.01,
+OPTIMIZATION_CONFIG = {
```

Comment:
(nit) I would always name test data such that is clear that it's just for tests, to avoid confusing it with real stuff .. `TEST_OPTIMIZATION_CONFIG` ?